packages/realm/scripts/ccache-clang++.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
I don't get why this is moved into the SDK?

Reply:
Good point.  I've moved this back into the root project scripts.